src/tests/Microsoft.PowerFx.Core.Tests/UserDefinedFunctionTests.cs

```diff
@@ -519,6 +519,30 @@ public void TestUDFsWithRestrictedTypes(string script)
             errors.AddRange(parseResult.Errors ?? Enumerable.Empty<TexlError>());
 
             Assert.Contains(errors, x => x.MessageKey == "ErrUDF_InvalidReturnType" || x.MessageKey == "ErrUDF_InvalidParamType");
+        }
+
+        [Theory]
+        [InlineData("Set(x: Number):Number = x + 1;", true)]
+        [InlineData("Count():Number = 5;", false)]
+        public void TestUDFsReservedNames(string script, bool expectedError)
```

Comment:
Since adding to this reserved list is a breaking change, can we have the list of reserverd names hardcoded in the test?  Ie, so if somebody adds a new reserved name, they need to update the test to allow it (this will help raise awareness that it's a breaking change. ) This is similar to what w edo with public surface tests.

Reply:
Good call